### PR TITLE
FAT-7654

### PR DESCRIPTION
**Pull request by nayimovag**
[FAT-7654](https://issues.folio.org/browse/FAT-7654)

### CODE REVIEW

File: cypress/e2e/bulk-edit/permissions/bulk-edit-csv-query-permissions.cy.js [hunk 1]
```diff
@@ -60,15 +60,15 @@ describe('bulk-edit', () => {
       });
       BulkEditSearchPane.verifySetCriteriaPaneSpecificTabs('Identifier', 'Query');
```
Comment (IhorBohdan) [inline cypress/e2e/bulk-edit/permissions/bulk-edit-csv-query-permissions.cy.js:61]: Remove waiter to the method

File: cypress/e2e/bulk-edit/permissions/bulk-edit-csv-query-permissions.cy.js [hunk 2]
```diff
@@ -60,15 +60,15 @@ describe('bulk-edit', () => {
...
+      BulkEditSearchPane.isUsersRadioChecked(false);
       BulkEditSearchPane.isDragAndDropAreaDisabled(true);
 
       BulkEditSearchPane.openQuerySearch();
       BulkEditSearchPane.verifySpecificTabHighlighted('Query');
       BulkEditSearchPane.usersRadioIsDisabled(false);
       BulkEditSearchPane.isBuildQueryButtonDisabled(false);
-
+      
+      BulkEditSearchPane.isUsersRadioChecked(false);
```
Comment (IhorBohdan) [inline cypress/e2e/bulk-edit/permissions/bulk-edit-csv-query-permissions.cy.js:71]: fix linter issues

File: cypress/e2e/bulk-edit/permissions/bulk-edit-in-app-query-permissions.cy.js [hunk 3]
```diff
@@ -59,17 +59,16 @@ describe('bulk-edit', () => {
       BulkEditSearchPane.verifySetCriteriaPaneSpecificTabs('Identifier', 'Query');
```
Comment (IhorBohdan) [inline cypress/e2e/bulk-edit/permissions/bulk-edit-in-app-query-permissions.cy.js:59]: Remove waiter to the method

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/bulk-edit/permissions/bulk-edit-csv-query-permissions.cy.js (hunk 1 above)
Comment (nayimovag) [reply cypress/e2e/bulk-edit/permissions/bulk-edit-csv-query-permissions.cy.js:59]: done

File: cypress/e2e/bulk-edit/permissions/bulk-edit-csv-query-permissions.cy.js (hunk 2 above)
Comment (nayimovag) [reply cypress/e2e/bulk-edit/permissions/bulk-edit-csv-query-permissions.cy.js:69]: fixed

File: cypress/e2e/bulk-edit/permissions/bulk-edit-in-app-query-permissions.cy.js (hunk 3 above)
Comment (nayimovag) [reply cypress/e2e/bulk-edit/permissions/bulk-edit-in-app-query-permissions.cy.js:57]: done

**Review by ostapwd — APPROVED** [review]

**Pull request merged**